README.md

```diff
@@ -59,6 +59,105 @@ style guide that you will not find in the GitHub one.
 
 ## Standards ##
 
+### Naming Conventions ###
+
+The Apple Swift [API design guidelines](https://swift.org/documentation/api-design-guidelines/) promote clarity over brevity. Code should be concise, readable and clear at the point of use. However, having compact code that sacrifices clarity is a non-goal.
+
+#### Types ####
+
+Capitalize all type names with camel case, while function and variable names start with a lower case.
```

Comment:
I changed this part with `Write all type names with UpperCamelCase, function and variable names with lowerCamelCase.`